src/logic/textBox.js

```diff
@@ -0,0 +1,72 @@
+import { answers, getAnswer} from './answer.js';
+import { getJoke, jokes } from './tellMeJoke';
+import { getFact, facts } from './tellMeFact';
+
+const displayTextBox = (text) => {
+  const Paragraph = document.createElement("p");
+  const questionParagraph = document.createElement("p");
+  const answerOutput = document.getElementById("answerOutput");
+ 
+
+  
+  questionParagraph.textContent = question;
```

Comment:
This code is not organised. As a developer I have to spend quite a while to read this. This is because you are mixing questionParagraph logis along with the Paragraph. Try to structure the 1 logic block under another, and seperate logic blocks with enter. 

Also i would suggest you making maby some funciton that would describe what are you exacly planning to do. Right now Paragrah.innerHTML = text and Paragrapsh.classList.add("answer-test") may be quite clear but in the future when you would make it more and more complicated it may take quite some time to get an idea of what you are planning to do here. So to avoid that maby try to create some kind of


const createAnswerParagraph = (answer) => {....}
const createQuestionParagrapsh = (question) => {...}

And then in the main code you would just have 
 answerOutput.appendChild(createAnswerParagraph(text));
 answerOutput.appendChild(createQuestionParagraph(question));

Reply:
I will like to discuss this issue with you on our next call.